dashboard/app/models/pd/workshop.rb

```diff
@@ -62,7 +62,7 @@ class Pd::Workshop < ApplicationRecord
     # organization.
     # Only current allowed values are "friday_institute" and nil.
     # "friday_institute" represents The Friday Institute,
-    # a regional partner whose model of virtual workshop was used
+    # a regional partner whose model of virtual workshop is being used
     # by several partners during summer 2020.
```

Comment:
Nit: maybe update to include 2021? Or just remove any year reference, maybe virtual is forever at this point :).

Reply:
😂great point- will do!